webservices/partition/base.py

```diff
@@ -316,6 +277,7 @@ def refresh_child(cls, cycle, queue_old, queue_new):
             # populating the child tables during normal partitioning.
             # Otherwise, an error is thrown due more values being specified
             # than there are columns to accept them.
+            # VM TODO - Is this still needed?
```

Comment:
Remove the `two_year_transaction_period` column from the `column_factory`...  You may be right!  With the partition creation happening on the SQL side now, we don't need to keep track of it anymore.  Just the ones that the queues don't already have we still need for the time being.